### ISSUE TITLE
Some mk2 adapter parts show asymmetrical lift and voxelization

### ISSUE THREAD

**Issue by ildifa** [dkavolis/Ferram-Aerospace-Research#56]
Some Mk2 adapter parts show a noticeable asymmetry in the lift produced, as can be seen from these pictures:

![Screenshot (64)](https://user-images.githubusercontent.com/48331910/57331457-eb290a80-7118-11e9-923b-9a8a7d348370.jpg)

![Screenshot (66)](https://user-images.githubusercontent.com/48331910/57331459-ebc1a100-7118-11e9-8ece-a066c361b2d2.jpg)

I guess this is due to an incorrect voxelization:
![Screenshot (65)](https://user-images.githubusercontent.com/48331910/57331458-ebc1a100-7118-11e9-8a86-8523534db73f.jpg)

This happens to me with Mk2 to 2.5m adapters, with the Mk2 inline cockpit, and also with Mk2 to Mk3 adapters (both straight and slanted). However, there is apparently no problem with 1.25m to Mk2 adapters.

**Comment by dkavolis** [comment]
Can you try adding
```
@PART[adapterMk3-Mk2]:AFTER[FerramAerospaceResearch]
{
	@MODULE[GeometryPartModule]
	{
		%forceUseMeshes = true
		%ignoreIfNoRenderer = true
	}
}
@PART[adapterMk3-Size2Slant]:AFTER[FerramAerospaceResearch]
{
	@MODULE[GeometryPartModule]
	{
		%forceUseMeshes = true
		%ignoreIfNoRenderer = true
	}
}
@PART[adapterMk3-Size2]:AFTER[FerramAerospaceResearch]
{
	@MODULE[GeometryPartModule]
	{
		%forceUseMeshes = true
		%ignoreIfNoRenderer = true
	}
}
@PART[adapterSize2-Mk2]:AFTER[FerramAerospaceResearch]
{
	@MODULE[GeometryPartModule]
	{
		%forceUseMeshes = true
		%ignoreIfNoRenderer = true
	}
}
```
to an MM patch? This will fix assymetrical voxelization. Inline cockpit looks to be voxelized symmetrically.

**Comment by ildifa** [comment]
I pasted it at the end of FARVoxelGeometryOverrides.cfg, and it fixed the bug. Fantastic! :)

![Screenshot (71)](https://user-images.githubusercontent.com/48331910/57562213-01d99680-7391-11e9-84bd-aea00dd647ef.jpg)
![Screenshot (72)](https://user-images.githubusercontent.com/48331910/57562214-02722d00-7391-11e9-89be-60df65b2a7a4.jpg)

I thought there was a problem with the direction of the cockpit's lift vector; now I realized that it was just a misunderstanding on my part...
Thanks for the fix!
